WordPressAuthenticator/Signup/SignupGoogleViewController.swift

```diff
@@ -141,6 +141,7 @@ private extension SignupGoogleViewController {
     /// Social Login Successful: Analytics + Pushing the Login Epilogue.
```

Comment:
We should probably also bump `.signedIn` from `socialSignupWasSuccessful`.  We bump both account_created  and signed_in events when creating a WordPress account via email magic link.  We should do the same for social flows.  (Tested to double check.)